Sources/HaCWebsiteLib/routes.swift

```diff
@@ -13,23 +13,16 @@ func getWebsiteRouter() -> Router {
 
   router.get("/", handler: LandingPageController.handler)
 
-  router.get("/workshops") { request, response, next in
-    defer {
-      next()
-    }
-    do {
```

Comment:
Turns out errors are automatically caught by Kitura: https://github.com/IBM-Swift/Kitura/commit/42eb6bf947f7ae49e22e16deedaad3935217729c

This means we can catch these errors and do something with them in a general-purpose error handling middleware. This is slated for #58

Reply:
Much nicer! 👍 as long as we're logging the Error

Reply:
Alas, we aren't yet. That's what the referenced issue (#58) is for. Is not logging a blocker?

Reply:
Nah, the issue is tagged appropriately so we Gucci